src/config/production.ts

```diff
@@ -34,7 +34,7 @@ const config: ConfigInterface = {
     release: process.env.SENTRY_RELEASE ?? throwError('No Sentry Release defined'),
   },
   telemetry: {
-    amplitudeKey: '968524573d0f8bd2e84460099bca9353'
+    amplitudeKey: '8fd6aa1d61c115e59e35b3adfd5dd41a'
```

Comment:
Is this the same as local? How did we said we could differentiate them if they are in the same dashboard? Or we do mot need to?

Reply:
yes it is the same as local. we can differentiate by environment